### PR TITLE
[4.12] Build ironic-agent-image on rhel9

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Review by iurygregory — APPROVED** [review]

**Comment by iurygregory** [comment]
I will add lgtm after CI result

**Comment by elfosardo** [comment]
this should not have merged, the configuration is wrong and needed review, my bad I didn't put it on hold :/

**Pull request merged**
